src/variant_detection/variant_output.cpp

```diff
@@ -72,72 +72,75 @@ void write_record(Cluster const & cluster,
 
     if (mate1.seq_name == mate2.seq_name)
     {
-        size_t const insert_size = cluster.get_average_inserted_sequence_size();
-        int const distance = mate2.position - mate1.position - 1;
-        int sv_length{};
-        int sv_length_iGenVar{};
-        std::string sv_type;
-
-        // Tandem Duplication
-        // In case of a small deletion inside of a duplication, the distance is a small positive value
-        if (cluster.get_common_tandem_dup_count() > 0 && static_cast<uint64_t>(distance) <= args.max_tol_deleted_length)
+        if (mate1.orientation == strand::forward)
         {
-            record.alt() = {"<DUP:TANDEM>"};
-            // Increment end by 1 because VCF is 1-based
-            record.info().push_back({.id = "END", .value = mate2.position + 1});
-            sv_length = distance + 2;
-            sv_length_iGenVar = insert_size;
-            sv_type = "DUP";
-        }
-        // Deletion OR Inversion
-        else if (distance > 0)
-        {
-            // Inversion
-            // An Inversion consists of 2 Breakpoints, thus it looks like a deletion with an inserted sequence
-            if (insert_size >= args.min_var_length)
+            size_t const insert_size = cluster.get_average_inserted_sequence_size();
+            int const distance = mate2.position - mate1.position - 1;
+            int sv_length{};
+            int sv_length_iGenVar{};
+            std::string sv_type;
+
+            // Tandem Duplication
+            // In case of a small deletion inside of a duplication, the distance is a small positive value
+            if (cluster.get_common_tandem_dup_count() > 0 && static_cast<uint64_t>(distance) <= args.max_tol_deleted_length)
             {
-                // Increment position by 1 because INV mate1 points on its last element
-                record.pos() += 1;
-                record.alt() = {"<INV>"};
+                record.alt() = {"<DUP:TANDEM>"};
                 // Increment end by 1 because VCF is 1-based
-                // Increment end by 1 because inversion ends one base before mate2 begins
                 record.info().push_back({.id = "END", .value = mate2.position + 1});
-                sv_length = distance;
-                sv_length_iGenVar = sv_length;
-                sv_type = "INV";
+                sv_length = distance + 2;
```

Comment:
why 2?

Reply:
I opened an issue for this https://github.com/seqan/iGenVar/issues/224